language/move-model/src/model.rs

```diff
@@ -2519,3 +2553,23 @@ impl<'env> FunctionEnv<'env> {
         )
     }
 }
+
+// =================================================================================================
+/// # Formatting
+
+impl fmt::Display for ConditionTag {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        use ConditionTag::*;
+        match self {
+            Requires => write!(f, "requires"),
+            Ensures => write!(f, "ensures"),
+            NegativeTest => write!(f, "negative-test"),
```

Comment:
Is NegativeTest for specifications that we expect to fail (similar to Kevin's project)?  If yes, I was expecting it to be a sub-classification of requires/ensures.

Reply:
Yes. But I expect to refactor this after the old backend is retired. There is a lot of mess right now how verification conditions are associated with assert/assume in the code (the ConditionTag and ConditionKind) which needs to be overhauled.